lib/matplotlib/font_manager.py

```diff
@@ -648,6 +649,12 @@ class FontProperties:
       'small', 'medium', 'large', 'x-large', 'xx-large' or an
       absolute font size, e.g., 12.
 
+    - mathtext_fontset: The family of fonts used in case this text contains
```

Comment:
"The family of fonts used to render math text; overrides :rc:`mathtext_fontset`.  Supported ..."